src/test/java/org/stratumproject/fabric/tna/behaviour/upf/MockP4RuntimeClient.java

```diff
@@ -0,0 +1,120 @@
+// Copyright 2022-present Intel Corporation
```

Comment:
same here even if it is a new file. I would not elevate ourselves to a special case - other organisations are not doing the same